english/your-dog/your-dog.md

```diff
@@ -1,3 +1,7 @@
-Then you wake up! Or did you really? But your dog is licking you in the face. Lord, that dog breath!
+Then you wake up! Or did you really? But your dog is licking you in the face. Lord, that dog breath!  Almost as fragrent as when he comes in from the rain.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.